Source/WebCore/contentextensions/SerializedNFA.h

```diff
@@ -45,27 +44,21 @@ class SerializedNFA {
     template<typename T>
     class Range {
     public:
-        Range(const T* begin, size_t size)
-            : m_begin(begin)
-            , m_size(size) { }
-        const T* begin() const { return m_begin; }
-        const T* end() const { return m_begin + m_size; }
-        size_t size() const { return m_size; }
-        const T* pointerAt(size_t i) const
-        {
-            RELEASE_ASSERT(i < m_size);
-            return begin() + i;
-        }
+        Range(std::span<const T> span)
+            : m_span(span)
+        { }
+        const T* begin() const { return std::to_address(m_span.begin()); }
+        const T* end() const { return std::to_address(m_span.end()); }
+        size_t size() const { return m_span.size(); }
+        const T* pointerAt(size_t i) const { return &m_span[i]; }
```

Comment:
Why doesn't this need to use `to_address`?

Reply:
I use `std::to_address()` to convert span iterators into raw pointers. Here I don't have an iterator, I have an element. I'm sure there is a way to write this using `std::to_address()` but I'm not clear why this would be better.